personalSite.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kanit:ital,wght@0,400;0,500;1,400&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body
+    style="
+      background-image: url(pawel-czerwinski-NJdfP9AlPjc-unsplash.jpg);
+      background-size: contain;
+      background-repeat: no-repeat;
+      font-family: 'Kanit', sans-serif;
+      width: 700px;
+    "
+  >
+    <main style="padding: 2rem">
+      <nav>
+        <ul style="display: flex; gap: 5rem; padding-left: 17rem">
+          <li>Contact</li>
+          <li><a href="#skills">Skills</a></li>
+        </ul>
+      </nav>
```

Comment:
Could be in a <head> element instead

Reply:
What du you mean? :slightly_smiling_face: What kind of element?

Reply:
The <nav> element could be in a <head> element above the <main>.

Reply:
Ah so! Thank you 🙂